### PR TITLE
Adding knitr

### PR DESCRIPTION
**Pull request by tomwright01**
Addressing issue #11, I have imported the material from @kbroman.

### CODE REVIEW

File: 15-knitr-markdown.Rmd [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+1. bold with double-asterisks
+1. italics with underscores
+1. code-type font with backticks
+```
+
+This will appear as:
+
+1. bold with double-asterisks
+1. italics with underscores
+1. code-type font with backticks
```
Comment (aammd) [inline 15-knitr-markdown.Rmd]: this part is clever! 😛

File: 15-knitr-markdown.Rmd [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+menu. Or you could put `pdf_document` or `word_document` in the header
+of the file.
+
+> ## Tip: Creating PDF documents {.callout}
+>
+> Creating .pdf documents may require installation of some extra software. If required this is detailed in an error message.
+>
+> Tex for windows is available [here](http://miktex.org/2.9/setup).
+>
+> Tex for mac is available [here](http://tug.org/mactex).
```
Comment (aammd) [inline 15-knitr-markdown.Rmd]: @tomwright01 Do you think it is worth discussing `spin()`, and the use of special comments that end up formatted as markdown (i.e. `#' a *markdown* comment`)? Or is that overload?

**Comment by aammd** [comment]
I love this. @tomwright01 how did you pull in Karl's work?

**Comment by tomwright01** [comment]
I can't take all (any) credit. I used the recipe [here](http://blog.neutrino.es/2012/git-copy-a-file-or-directory-from-another-repository-preserving-history/).

Seems it doesn't work in every edge case, but did work here. Any suggestions where this could be documented as it seems to be a fairly common task?

**Pull request merged**